src/main/resources/ui/controller/sensor/TimeController.js

```diff
@@ -29,8 +29,15 @@ class TimeController {
         let cronExpressionPath = "/workflow/sensor/properties/settings/variables/cronExpression";
         let cronExpression = this._model.getProperty(cronExpressionPath);
         if(!cronExpression) {
-            this._model.setProperty(cronExpressionPath, "0 0 * * * *")
+            this._model.setProperty(cronExpressionPath, "0 0 0 ? * * *")
```

Comment:
Did you change this intentionally from a 6-character to a 7-character cron expression?

Reply:
Yes, 7th character is optional, but the expression is still valid.